fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMaker.java

```diff
@@ -57,33 +81,29 @@ public ArithmeticMaker(
      * @param function  The arithmetic operation performed by the LogicalMetrics constructed by this maker
      */
     public ArithmeticMaker(MetricDictionary metricDictionary, ArithmeticPostAggregationFunction function) {
-        // TODO: Deprecate me, mappers should always be specified at creation time, not implicitly
-        this(metricDictionary, function, new SketchRoundUpMapper());
+        this(
+                metricDictionary,
+                function,
+                (Function<String, ResultSetMapper>) (ignore) -> NO_OP_MAPPER
```

Comment:
Definitely need to call this change out in the ChangeLog, and I'm pretty sure there are some that will need to change due to this.